mention -target in destroy docs (#32615)
hashicorp/terraform: website/docs/cli/commands/destroy.mdx

-> **Note:** The `-destroy` option to `terraform apply` exists only in
Terraform v0.15.2 and later. For earlier versions, you _must_ use
`terraform destroy` to get the effect of `terraform apply -destroy`.

> ## Options
> 
> In exceptional circumstances, it may be useful to use the `-target` option. 
> This will destroy only a particular resource (and its dependencies):
> 
> ```
> terraform destroy -target RESOURCE_TYPE.NAME
> ```
> 
> For more about `-target`, see [`plan`](/cli/commands/plan).

Reviewer: ```suggestion
### Target a specific resource

You can use the `-target` option to destroy a particular resource and its dependencies:

```
terraform destroy -target RESOURCE_TYPE.NAME
```

Refer to [Resource Targeting](/terraform/cli/commands/plan#resource-targeting) for additional information about using the `target` option.
```
We should use a more specific heading because "Options" might give the impression that `target` is the only option you can use. If you're scanning the article, you can easily miss the line about being able to use the same options as the `apply` command. Also, describing this as a specific use-case means we should also position it as a subtopic of "Usage".

It might be hard for other users know what qualifies as an "exceptional circumstance" and when to use the option. To reduce potential confusion, let's just describe the functionality.  

Per our style guide, use active voice by addressing users in the second person and avoid using parentheses.
